src/main/java/net/dv8tion/jda/core/requests/restaction/RoleAction.java

```diff
@@ -43,7 +43,7 @@
 {
 
     protected final Guild guild;
-    protected long permissions = 0;
+    protected Long permissions;
```

Comment:
I don't understand the decision behind this change.
Additionally, there is no way to `null`-ify this value after it has been set.

Reply:
It makes a difference whether you send permissions 0 or not at all. I made this nullable to allow using the default values (everyone role permissions)

Reply:
Then you need to make it possible to actually set the Permissions to null then. You didn't modify any of the setters to support this.